frontend/device/android/device.lua

```diff
@@ -486,10 +481,12 @@ end
 function Device:install()
     local UIManager = require("ui/uimanager")
     local ConfirmBox = require("ui/widget/confirmbox")
+    local Event = require("ui/event")
```

Comment:
Definitely doesn't need to come after ConfirmBox. ;-)

Reply:
https://github.com/koreader/koreader-base/wiki/Coding-style#order-of-requires

from top modules to specific modules  (following the alphabet whenever possible)

```lua
local Event = require("ui/event")
local UIManager = require("ui/uimanager")
local ConfirmBox = require("ui/widget/confirmbox")
```
strictly alphabetical

```lua
local ConfirmBox = require("ui/widget/confirmbox")
local Event = require("ui/event")
local UIManager = require("ui/uimanager")

```

Reply:
choose your own adventure :smile:

Reply:
Our style guide specifies alphabetical. ;-)

https://github.com/koreader/koreader-base/wiki/Coding-style

Reply:
Then I'm afraid I didn't understand your comment

> Definitely doesn't need to come after ConfirmBox. ;-)

Reply:
I thought you might've had a reason (e.g., ConfirmBox uses UIManager so the order might matter), which doesn't apply to Event. But apparently you simply added it at the bottom with no rationale. ^_^